cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleServiceTest.java

```diff
@@ -757,6 +752,9 @@ public void testUpdateSourceControlMetaWithNonExistingApp() throws Exception {
     Assert.assertNotNull(updatedDetail.getSourceControlMeta());
     Assert.assertEquals("updated-file-hash", updatedDetail.getSourceControlMeta().getFileHash());
     Assert.assertEquals("updated-commit-id", updatedDetail.getSourceControlMeta().getCommitId());
+
+    deleteAppAndData(new ApplicationId(TEST_NAMESPACE1, AllProgramsApp.NAME,
```

Comment:
why this change?

Reply:
While running ApplicationLifecycleServiceTest, in this specific test we are deploy an app and update its source control metadata. In another test, testUpdateSourceControlMeta, we deploy the same app. Its filehash and commitId becomes null but the last modified retains (it is the expected behaviour now), hence source control metadata is not null. 
I made this change so that a unit test doesn't affect another test.